src/_imaging.c

```diff
@@ -533,7 +536,7 @@ getink(PyObject* color, Imaging im, char* ink)
                 return NULL;
             }
         } else {
-            PyErr_SetString(PyExc_TypeError, "color must be int");
+            PyErr_SetString(PyExc_TypeError, "color must be int or tuple");
```

Comment:
```python
>>> PIL.Image.new('I', (200,100), (255,255,255))
...
TypeError: color must be int or tuple
```
is not very helpful.

The following suggestion would avoid confusion if a user specifies a tuple with two or more values:
```suggestion
            PyErr_SetString(PyExc_TypeError, "color must be int or single-element tuple");
```

The `IMAGING_TYPE_UINT8` branch provides a more specific error message for an invalid tuple below (from `PyArg_ParseTuple`).